### ISSUE TITLE
Support CUDA 11.5 and TensorRT 8.2.1.3?

### ISSUE THREAD

**Issue by hxcai** [Tencent/TPAT#20]
Thanks for excellent work! I want to know if this tools can work for CUDA 11.5 and Tensorrt 8.2.1.3?

**Comment by buptqq** [comment]
Yes, Although our test environment is CUDA-11.0 and TesnorRT 8.0.0.3. 
But the interface of Plugin in TensorRT 8.x are common.
